src/Context/Response/Customer/CustomerResponseStruct.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\App\SDK\Context\Response\Customer;
+
+use Shopware\App\SDK\Context\Response\ResponseStruct;
+
+class CustomerResponseStruct extends ResponseStruct
+{
+    public ?string $title;
+
+    /**
+     * @var 'private'|'business'|null
```

Comment:
enum? :see_no_evil:

Reply:
I do not see, that this is appropriate
1. We do not use any enums for account types in platform
2. There are only two possibilities and we will keep it that way

Reply:
was just asking :innocent: if a string only allows certain values, the question for enum is quite obvious and as it is a quite new feature of PHP, it is often forgotten :grin: